agent-manager/.env.example

```diff
@@ -7,6 +7,10 @@ CARDANO_NETWORK_MAGIC=4
 
 KUBER_BASE_URL=
 KUBER_API_KEY=
+BLOCKFROST_API_KEY=
+BLOCKFROST_BASE_URL=
```

Comment:
Lets use the NETWORK= preview|preprod|sanchonet|mainnet to directly determine BLOCKFROST_BASE_URL.

Reply:
Done